app/views/shared/nav/_nav_links.html.haml

```diff
@@ -23,7 +23,7 @@
 
 -# .visible-with-nav is visible > 979px
 = form_tag global_search_path, class: "navbar-search pull-right visible-with-nav" do
-  = text_field_tag :search, nil, placeholder: t("global_search.search"), class: "search-query span2"
+  = text_field_tag :search, nil, placeholder: t("global_search.search"), class: "search-query span2", "aria-label": "navSearch"
```

Comment:
Should the label match the `placeholder`, or is this a reference to another element?